GUI/UI.cs

```diff
@@ -350,8 +350,8 @@ public static class UI {
                 DrawMiniScores(miniScoresPos, Game.scores);
                 DrawLevelName(levelNamePos, ColorScheme.black);
                 LevelMetadata metadata = Map.currentLevel.metadata;
-                if(metadata.skippable && Game.music.PlayingOffset.AsMilliseconds() < metadata.skipTime && skipButton.Draw()) {
-                    Game.music.PlayingOffset = Time.FromMilliseconds(metadata.skipTime);
+                if(metadata.skippable && Game.music.PlayingOffset.AsMilliseconds() - metadata.initialOffsetMS < metadata.skipTime && skipButton.Draw()) {
+                    Game.music.PlayingOffset = Time.FromMilliseconds(metadata.skipTime) + Time.FromMilliseconds(metadata.initialOffsetMS);
```

Comment:
It's faster to add the two values together (`skipTime` and `initialOffsetMS`) and the convert them, than convert both of them separately and then add together